include/crm/common/ipc.h

```diff
@@ -159,27 +159,33 @@ int pcmk_ipc_purge_node(pcmk_ipc_api_t *api, const char *node_name,
  * Generic IPC API (to eventually be deprecated as public API and made internal)
  */
 
-/* *INDENT-OFF* */
 enum crm_ipc_flags
 {
-    crm_ipc_flags_none      = 0x00000000,
-
-    crm_ipc_compressed      = 0x00000001, /* Message has been compressed */
-
-    crm_ipc_proxied         = 0x00000100, /* _ALL_ replies to proxied connections need to be sent as events */
-    crm_ipc_client_response = 0x00000200, /* A Response is expected in reply */
+    crm_ipc_flags_none              = UINT32_C(0),
+    //! Message has been compressed
+    crm_ipc_compressed              = (UINT32_C(1) << 0),
+    //! _ALL_ replies to proxied connections need to be sent as events
+    crm_ipc_proxied                 = (UINT32_C(1) << 8),
+    //! A response is expected in reply
+    crm_ipc_client_response         = (UINT32_C(1) << 9),
 
     // These are options for Pacemaker's internal use only (pcmk__ipc_send_*())
-    crm_ipc_server_event    = 0x00010000, /* Send an Event instead of a Response */
-    crm_ipc_server_free     = 0x00020000, /* Free the iovec after sending */
-    crm_ipc_proxied_relay_response = 0x00040000, /* all replies to proxied connections are sent as events, this flag preserves whether the event should be treated as an actual event, or a response.*/
+
+    //! Send an Event instead of a Response
+    crm_ipc_server_event            = (UINT32_C(1) << 16),
+    //! Free the iovec after sending
+    crm_ipc_server_free             = (UINT32_C(1) << 17),
+    //! All replies to proxied connections are sent as events.  This flag
+    //! preserves whether the events should be treated as an Event or a Response
+    crm_ipc_proxied_relay_response  = (UINT32_C(1) << 18),
 
 #if !defined(PCMK_ALLOW_DEPRECATED) || (PCMK_ALLOW_DEPRECATED == 1)
-    crm_ipc_server_info     = 0x00100000, //!< \deprecated Unused
-    crm_ipc_server_error    = 0x00200000, //!< \deprecated Unused
+    //!< \deprecated Unused
```

Comment:
The `<` is for same-line comments; remove here and below (not that it's particularly important)

Reply:
Might as well do it right.